packages/kbn-es/src/install/install_source.ts

```diff
@@ -84,15 +84,15 @@ async function sourceInfo(cwd: string, license: string, log: ToolingLog = defaul
   log.info('on %s at %s', chalk.bold(branch), chalk.bold(sha));
   log.info('%s locally modified file(s)', chalk.bold(status.modified.length));
 
-  const etag = crypto.createHash('md5').update(branch); // eslint-disable-line @kbn/eslint/no_unsafe_hash
+  const etag = crypto.createHash('sha256').update(branch);
```

Comment:
Self Review: I think these can be changed (yarn es source worked) and it would be nice to remove all md5 possible. Thoughts @jbudz @Ikuni17 .

Reply:
lgtm